pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: apptive_grid_flutter_workspace
 
 environment:
- sdk: '>=2.18.0 <3.0.0'
+ sdk: '>=3.0.0 <4.0.0'
 
 dev_dependencies:
- melos: ^3.0.0
+ melos: ^3.1.1
```

Comment:
These changes also don't seem to be necessary

Reply:
reverted